chdb-parquet/results/c6a.metal.json

```diff
@@ -1,55 +1,55 @@
+    "date": "2024-08-12",
     "machine": "c6a.metal, 500gb gp2",
     "cluster_size": 1,
     "comment": "",
     "tags": ["C++", "column-oriented",  "ClickHouse derivative", "embedded", "stateless", "serverless"],
     "load_time": 0,
     "data_size": 14737670832,
     "result": [
-[0.18159093600002052,0.03465310399997179,0.03535941499990258],
-[0.2831564299999627,0.07290863800017178,0.07088584300004186],
-[0.4106436740000845,0.11962774199992054,0.11457044600001609],
-[0.47319400800006406,0.11146962500015434,0.11445495399993888],
-[1.6001970140000594,1.1832852380000531,1.2053231469999446],
-[1.9747222290000082,1.3993265449998944,1.4632436899998993],
-[0.293016236000085,0.10769365200007996,0.10403609099989808],
-[0.30888328800006093,0.08704660199987302,0.08690985099997306],
-[0.9676274400001148,0.4142185770001561,0.4095282320001843],
-[1.508716829999912,0.4386639290000858,0.42993967000006705],
-[0.7211182660000759,0.22335372600014125,0.22127814999998918],
-[0.8024834689999807,0.24581066700011434,0.2369266250000237],
-[0.9941888450000533,0.34986996500015266,0.36299695699995027],
-[2.2883914159999676,0.5018040939999082,0.39797729199995047],
-[1.0599523600001248,0.39205417400012266,0.3990406109999185],
-[0.614120850999825,0.26384819800000514,0.27866645900007825],
-[2.468679433000034,0.7421945380001489,0.707125589000043],
-[2.390665940999952,0.6761940030000915,0.6212907079998331],
-[4.531502312999919,1.2391869979999228,1.2929706920001536],
-[0.41148081200003617,0.08244017399988479,0.07748579300005076],
-[9.470291036999924,0.4142645150000135,0.42491283499998644],
-[11.166745869999886,0.4193407879999995,0.47778978899987123],
-[21.716601035000167,0.8906946930001141,0.9057610759998624],
-[55.87572047799995,4.693745457999967,4.408192557999882],
-[2.6822018080001726,0.2497673709999617,0.18029253800000333],
-[0.9117738509999072,0.19944890400006443,0.19231710099984411],
-[2.6458359000000655,0.19551512300017748,0.17991257200014843],
-[9.665634246000081,0.644791166999994,0.5677360409999892],
-[8.823865798999805,1.8234754839998004,1.911617729999989],
-[0.887859853000009,0.7314614570000231,0.7472309880001831],
-[2.609201996000138,0.3776117290001366,0.42668743700005507],
-[6.085526299000094,0.5356784809998771,0.5014943220000987],
-[5.348554426000192,1.4791098619998593,1.6584785480001756],
-[9.919410929999913,1.198197147999963,1.3908020020001004],
-[9.918529573000114,1.4790268179999657,1.2929598839998562],
-[0.5918198439999287,0.36387553300005493,0.3893138729999919],
-[0.353354839000076,0.15782711399992877,0.17798462199993992],
-[0.3100515420001102,0.16545474899999135,0.10927909599990926],
-[0.3464544030000525,0.19426173600004404,0.10704828600000837],
-[0.5001845439999215,0.28064504200006013,0.28905982800006313],
-[0.3150378160000855,0.12564524300000812,0.08628042499981348],
-[0.24802713799999765,0.1437892629999169,0.0803308869999455],
-[0.2594364960000348,0.2181738829999631,0.08233647700012625]
+[0.18839595599996528,0.05121623700000555,0.04376224599997158],
```

Comment:
There are also chDB results for c6a.4xlarge (`c6a.4xlarge.json`) which are not updated. Consider updating them as well, or removing the old measurements (to avoid confusing).

Reply:
There is some strange result on c6a.4xlarge, investigating.